src/lib/core/core.ts

```diff
@@ -82,12 +82,6 @@ export {
   A11yModule,
 } from '@angular/cdk/a11y';
 
-export {
-  UniqueSelectionDispatcher,
-  UniqueSelectionDispatcherListener,
-  UNIQUE_SELECTION_DISPATCHER_PROVIDER,
-} from './coordination/unique-selection-dispatcher';
-
```

Comment:
For backwards-compat, this should continue re-exporting the symbols from the CDK (see the `A11yModule` above).

Reply:
Will do